### PR TITLE
Add Enzyme documentation

### PR DESCRIPTION
**Pull request by jmerle**
Added a scraper for the Enzyme documentation ([Trello card](https://trello.com/c/dWNfdGOR)).

The documentation is licensed under the [MIT license](https://github.com/airbnb/enzyme/blob/master/LICENSE.md).

Enzyme doesn't have a logo, so the default icon should be used.

### CODE REVIEW

**Comment by jgornick** [comment]
@Thibaut There are a couple of PRs out there that I would love to see available on devdocs.io. Some of them have been around since Oct. 2018 (this one). 

I was just curious if there is anything that's holding these back from being merged. I'm assuming time is the primary reason 😄 

I absolutely love devdocs.io and thanks for all the awesome work!

**Comment by jmerle** [comment]
Yup, time it is. Thibaut is almost done with moving everything DevDocs related over to freeCodeCamp. As far as I know, the last things to do at the moment are moving over all accounts from different services (Sentry, Heroku etc.) to freeCodeCamp and giving permissions for merging/deploying to the correct people.

Besides that, freeCodeCamp has hired @ahmadabdolsaheb to work on DevDocs full-time, and both @j-f1 and myself will receive the permissions to review (and possibly merge/deploy). This should speed up the PR process a lot, since it would mean Thibaut is no longer required to make time to review/merge/deploy contributions.

Although I can't say for certain when the move will be completed, I am hoping for it to be done by the end of this month.

**Comment by Thibaut** [comment]
freeCodeCamp has all the accounts now. They're ramping up on the code and deploy process.

**Comment by jgornick** [comment]
Any updates on getting this merged? Thanks!

**Comment by j-f1** [comment]
The freeCodeCamp folks are working on something to do with their main product, but they’ll be putting more time into this once that’s done.

**Comment by jgornick** [comment]
@j-f1 and @jmerle Just checking in again to see if there are any updates on getting the PRs for new documentations live on devdocs.io? Is freeCodeCamp looking for help with the deployment or development of this?

**Comment by j-f1** [comment]
We’re still waiting on the freeCodeCamp folks at the moment.

**Comment by jmerle** [comment]
> @j-f1 and @jmerle Just checking in again to see if there are any updates on getting the PRs for new documentations live on devdocs.io?

The plan is to get the non-scraper PR's merged and deployed asap (basically all PR's labeled [improvement](https://github.com/freeCodeCamp/devdocs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Aimprovement) and [feature](https://github.com/freeCodeCamp/devdocs/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+label%3Afeature+)). We require one review per PR, and I'll get my reviews on PR's that are not mine done by the end of this week (got to do some performance testing on #714 and check if #724 is correct). I don't know the status of @j-f1 reviewing my PR's.

After those "minor" (mostly minor) PR's are merged we can get started with the scraper PR's, which take a bit more work to merge and deploy correctly.

> Is freeCodeCamp looking for help with the deployment or development of this?

PR's and feature requests are always welcome ofcourse, although it might take a while for it to get merged. As far as adding new maintainers to the project goes, I think it's not wise to add new maintainers until we've solved the PR backlog that we have at the moment.

> We’re still waiting on the freeCodeCamp folks at the moment.

True to some degree, although I think the best thing to do right now is review all PR's that need to be deployed first, instead of waiting on @QuincyLarson for S3 and Heroku related permissions/accounts. After we got the PR's lined up, it shouldn't be much work for Quincy to provide us the accounts that we need to have access to in order to deploy.

File: lib/docs/filters/enzyme/clean_html.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        css('pre').each do |node|
+          cls = node.at_css('code')['class']
+
+          unless cls.nil?
+            node['data-language'] = cls.split('-')[1]
+          end
+
+          node.content = node.content
+        end
+
```
Comment (j-f1) [inline lib/docs/filters/enzyme/clean_html.rb]: Can you remove the badges from http://localhost:9292/enzyme/?

Comment (j-f1) [inline lib/docs/filters/enzyme/clean_html.rb]: Can you remove the badges from http://localhost:9292/enzyme/?

File: lib/docs/scrapers/enzyme.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    self.release = '3.10.0'
+    self.base_url = 'https://airbnb.io/enzyme/'
+    self.root_path = 'index.html'
+    self.links = {
+      code: 'https://github.com/airbnb/enzyme'
+    }
+
+    html_filters.push 'enzyme/entries', 'enzyme/clean_html'
+
+    options[:skip] = %w(CHANGELOG.html docs/future.html CONTRIBUTING.html)
```
Comment (j-f1) [inline lib/docs/scrapers/enzyme.rb]: Can you also ignore `docs/guides.html` and `docs/api/index.html`? It’s pretty much a duplicate of the regular Guides type page.

```suggestion
    options[:skip] = %w(CHANGELOG.html docs/future.html docs/guides.html docs/api/index.html CONTRIBUTING.html)
```

I wish there was a way to specify custom type pages…

File: lib/docs/filters/enzyme/clean_html.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    class CleanHtmlFilter < Filter
+      def call
+        @doc = at_css('.page-inner > section')
+
+        # Clean headers
+        css('h1').each do |node|
+          node.content = node.content
+        end
+
+        # Make headers on reference pages bigger
```
Comment (j-f1) [inline lib/docs/filters/enzyme/clean_html.rb]: Can you also increase h3s to h2s in the API docs? http://localhost:9292/docs/api/selector

**Pull request merged**